.github/workflows/ci.yml

```diff
@@ -33,6 +33,8 @@ jobs:
             ./target
           key: ${{ runner.os }}-cargo-${{ matrix.platform.target }}-${{ hashFiles('**/Cargo.toml') }}
           restore-keys: ${{ runner.os }}-cargo-${{ matrix.platform.target }}-${{ hashFiles('**/Cargo.toml') }}
+      - uses: Vampire/setup-wsl@v1
```

Comment:
I suppose we need this for `686-pc-windows-msvc`?

Reply:
I saw this workaround to our issue: https://github.com/dudykr/stc/blob/64bf11b206d5e93ea9901022ab91273574f61c28/CONTRIBUTING.md#troubleshooting

Let me try to install `cargo` here and see if this fixes everything